db/migrate/20240412103303_add_worldwide_organisation_page_translations_table.rb

```diff
@@ -0,0 +1,9 @@
+class AddWorldwideOrganisationPageTranslationsTable < ActiveRecord::Migration[7.1]
+  def up
+    WorldwideOrganisationPage.create_translation_table! title: :string, summary: :text, body: :text
```

Comment:
These fields should be able to be removed from the `worldwide_organisation_pages` at this point.